content/blog/2023/02/2023-02-27-build-msi-locally.adoc

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: "How to build an unsigned Jenkins MSI on your Windows machine"
+tags:
+- jenkins
+- windows
+- msi
+authors:
+- gounthar
+discourse: true
+---
+
+Should you ever need to rebuild a Jenkins MSI on your Windows machine, here is a way to do it.
+
+== Pre-requisites
+
+=== Jenkins WAR file
+
+First of all, you should get the Jenkins war file that will be inside that MSI file.
+You can get it from the official Jenkins website or from https://updates.jenkins.io/[the Jenkins update center].
+
+Check https://www.jenkins.io/download/[Jenkins download page] and download the latest weekly version of Jenkins for example.
+You could always hit https://updates.jenkins.io/latest/jenkins.war[the direct link] to get the latest weekly version, but then you wouldn't know which version number you are using. Just saying.
+
+=== Git
+
+There are quite a few ways to get Git on Windows, but the most straightforward is to see what the https://git-scm.com/download/win[official Git website recommends].
+You have a few other options here:
+
+- Install https://community.chocolatey.org/packages/git[Git for Windows]
+- Use your IDE to clone the repository
+- Use https://desktop.github.com/[GitHub Desktop]
+- Use https://www.sourcetreeapp.com/[SourceTree]
+- Use Cygwin integrated https://cygwin.com/packages/summary/git.html[Git command]
```

Comment:
Similar to the packaging docs, I think listing one way to obtain git is enough.

Reply:
At least I tried. 🤷 🤣